presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -190,26 +188,24 @@ private static String hiveTableProperties(TransactionalTableType transactionalTa
 
     private static void compactTableAndWait(CompactionMode compactMode, String tableName, String partitionString, Duration timeout)
     {
-        Set<String> existingCompactionsIds = getTableCompactions(compactMode, tableName)
-                .map(row -> row.get("compactionid"))
-                .collect(toUnmodifiableSet());
-
         onHive().executeQuery(format("ALTER TABLE %s %s COMPACT '%s'", tableName, partitionString, compactMode.name()));
 
-        Set<String> tableCompactionsIds = Sets.difference(getTableCompactions(compactMode, tableName)
-                .map(row -> row.get("compactionid"))
-                .collect(toUnmodifiableSet()),
-                existingCompactionsIds);
+        // Since we disabled table auto compaction we can assume that every compaction is triggered in this test
```

Comment:
> Simplify compaction logic in TestHiveTransactionalTabl

What does this commit change?
I would prefer for this code to be defensive, especially that it's based on assumptions and hard to test.
E.g. if we want to assume there is no compaction, we should _verify_ there initially is no compaction.
(or leave as it was)

nit: typo in cmt

Reply:
I'll add an assertion that there are no compactions